spec/models/review_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Review, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
en los tests de modelos debería siempre al menos haber un test que prueba que la factory esté bien definida. Puedes ver un [ejemplo en el purchase](https://github.com/platanus/the-store2/blob/master/spec/models/purchase_spec.rb). De hecho, en este caso me tinca que tiraría error, ya que `user` y `item` están como `nil` en la factory, y por defecto el `belongs_to` los deja como requeridos